### PR TITLE
Add package installation first page

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: content/doc/installation/from-package.en-us.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  sidebar:
+    parent: "installation"
+    name: "From package"
+    weight: 20
+    identifier: "install-from-package"
+---
+
+# Installation from package
+
+Currently we only support `brew` if you are using Mac OS X.
```
Comment (bkcsoft) [inline content/doc/installation/from-package.en-us.md]: Mac OS X or macOS? 😉

Comment (tboerger) [reply content/doc/installation/from-package.en-us.md]: macOS

Comment (lunny) [reply content/doc/installation/from-package.en-us.md]: done.

**Comment by bkcsoft** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Pull request merged**